src/merge_maps_kinetic.cpp

```diff
@@ -44,13 +44,13 @@ void MergeMapTool::SetConfigs()
   {
     resolution_ = 0.05;
   }
-
   sstS_.push_back(nh_.advertise<nav_msgs::OccupancyGrid>("/map", 1, true));
   sstmS_.push_back(nh_.advertise<nav_msgs::MapMetaData>( \
                                                     "/map_metadata", 1, true));
-  ssMap_ = nh_.advertiseService("merge_maps", 
+  ros::NodeHandle nh;
+  ssMap_ = nh.advertiseService("merge_maps",
                                         &MergeMapTool::MergeMapCallback, this);
-  ssSubmap_ = nh_.advertiseService("add_submap", 
+  ssSubmap_ = nh.advertiseService("add_submap",
```

Comment:
Why are these not using the local nodehandle? There needs to be separate services for loading maps for merging vs normal. These cannot be interchangable